README.md

```diff
@@ -72,12 +72,9 @@ This starts a simple React site with the main Discovery UI component as the sole
 
 `npm run stop`
 
-## Build Package
-
-`npm run build:library`
-
 ## Using the Package Locally
```

Comment:
is a version bump required anywhere for react_discovery_ui?

Reply:
I think package.json needs to be bumped, then the new version needs to be referenced in discovery_ui

Reply:
Good catch. 

@c-m-duncan can you add a pull request template to this repo that asks for a package.json version bump?

That will be required for a new version to be published correctly to NPM after this is merged and we cut a release.